src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -390,6 +390,15 @@ void ICACHE_RAM_ATTR SX127xDriver::TXnb(uint8_t * data, uint8_t size, SX12XX_Rad
   // }
   SetMode(SX127x_OPMODE_STANDBY, SX12XX_Radio_All);
 
+  if (radioNumber == SX12XX_Radio_NONE)
+  {
+      radioNumber = lastSuccessfulPacketRadio;
+  }
```

Comment:
This function is not called if radio is `SX12XX_Radio_NONE`.
See PR #2155
It cal also be set to `SX12XX_Radio_All` for gemini mode so should probably increment telem_count on both radios in that case.
This would also hold for the SX1280 code too.

Reply:
Feel free to correct the branch directly, if you wish.
I'll be away for several hours, or all day long.